src/audio/asrc/asrc.c

```diff
@@ -987,14 +987,14 @@ static int asrc_copy(struct comp_dev *dev)
 	sink = list_first_item(&dev->bsink_list, struct comp_buffer,
 			       source_list);
 
-	source = buffer_acquire_irq(source);
-	sink = buffer_acquire_irq(sink);
+	source = buffer_acquire(source);
```

Comment:
Why has the IRQ and non IRQ become equivalent now ? There should be no changes for xtos.

Reply:
@lgirdwood because #5286 makes all spin-locks disable local IRQs. The first two commits in this PR are actually from there. Let's discuss there if you have any further questions

Reply:
Ok, I noticed the spinlock change and thought you were deprecating #5286. However, #5286 has the context now that's missing here.